### ISSUE TITLE
"minimal" image should not come with additional packages installed

### ISSUE THREAD

**Issue by leonklingele** [ayufan-rock64/linux-build#379]
Additional packages such as `iperf3`, `wget`, `build-essential`, `figlet`, `nano`, `git`, etc. should not be installed on a "minimal" image. Those can be installed later if required but should not land there per default.

**Comment by ayufan** [comment]
Why? For me definition of minimal is without a GUI :)

**Comment by leonklingele** [comment]
Yeah, but why install those packages by default which will only bloat the system? Everyone using any of those tools will probably know about `apt install` :)

**Comment by ayufan** [comment]
For conscience of testing minimal images.

**Comment by leonklingele** [comment]
What do you think of supporting another image type, e.g. `-minimal-stripped` which will not include those packages?

> For conscience of testing minimal images.

Why not install additional packages automatically/manually in your testing env?

**Comment by ayufan** [comment]
I don’t think so. You can uninstall them. The need for additional variant
like core is negligible it seems.

**Comment by ahmedmagdiosman** [comment]
IMO, this is a dev SBC. The tools mentioned are very handy to have and consume negligible amount of resources. I was actually happily surprised to find git & iperf3 installed by default.

**Comment by DmitryMyadzelets** [comment]
This is not really an "issue" for such a thing as an image, and should be closed.
